### PR TITLE
Sub scale lookup table

### PR DESCRIPTION
**Pull request by mtg137**
# Resolves [94](https://app.zenhub.com/workspaces/mindlogger-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-applet-builder/94)

### CODE REVIEW

**Review by Matchdev120 — CHANGES_REQUESTED** [review]
Please fix these merge conflicts.

**Comment by mtg137** [comment]
I fixed merge conflicts.

**Review by Matchdev120 — APPROVED** [review]
LGTM!

**Comment by devbtech** [comment]
@mtg137 Please bump up version to `0.14.33`

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.14.29] - 2020-12-21
 - Fixed media items not working
 
+## [0.14.29] - 2020-12-21
```
Comment (devbtech) [inline CHANGELOG.md:17]: Please update changelog as well.

Comment (mtg137) [reply CHANGELOG.md:17]: I already updated it.

Comment (mtg137) [reply CHANGELOG.md:17]: https://github.com/ChildMindInstitute/mindlogger-app/pull/1395/commits/2ba1ddb8b55f2ef2c8a165669d7f21fe31f6022f

Comment (devbtech) [reply CHANGELOG.md:17]: Your PR was being delayed until it's reviewed and other PRs are already merged with later versions. You need to update the changelog description with the right version and put it in right place.

Comment (mtg137) [reply CHANGELOG.md:17]: hm, I'll update it.
should we update version and changelog several times like now to merge one PR?

Comment (mtg137) [reply CHANGELOG.md:17]: It's third time I update the version/changelog ...

**Pull request merged**